### PR TITLE
Master branch debug

### PR DESCRIPTION
**Pull request by micheelengronne**
# Description:

Debugging the master branch.

The idea is to be able to run all tests locally. Currently, sqlite is buggy.
______________

# Tasks:

-  Describe the problem / feature
-  Write tests

I will abide by the [code of conduct](https://github.com/gemstash/gemstash/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by olleolleolle** [comment]
@deivid-rodriguez Would it be beneficial to think about skipping the sqlite3 support? Are people using that? Maybe they _are_?

**Comment by olleolleolle** [comment]
@micheelengronne Thanks for the 🔍 of this issue! 

Some of the commits could be their own PRs!

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+.yardoc
+Gemfile.lock
+_yardoc/
+coverage/
+doc/
+pkg/
+spec/reports/
+tmp/
+html/
+lib/gemstash/man/
```
Comment (olleolleolle) [inline .gitignore:11]: https://git-scm.com/docs/gitignore seems to describe the initial slash, as well.

That is, in the last example.

**Comment by micheelengronne** [comment]
Indeed. But we should coordinate with @bronzdoc We should work on that on saturday the 16th at 16.00 UTC on slack. Do you join ?

**Comment by deivid-rodriguez** [comment]
@olleolleolle I'm not sure but I'm glad you and @bronzdoc are trying to improve things here. This repo needs a lot of ❤️.

**Pull request closed without merging**